lib/tasks/shared.rake

```diff
@@ -0,0 +1,6 @@
+# Allow time for AppSignal to ship logs before the process exits
+# https://docs.appsignal.com/ruby/integrations/rake.html#rake-tasks-and-containers
+def wait_for_logs_to_flush
+  Appsignal.stop "rails"
+  sleep 5
+end
```

Comment:
I have a similar helper in email.rake - do you think you could consolidate that copy into this one?

Reply:
Oh totally, I hadn't noticed we already had something in place for this.